cmd/generic-handlers.go

```diff
@@ -139,24 +139,23 @@ func setBrowserCacheControlHandler(h http.Handler) http.Handler {
 	return cacheControlHandler{h}
 }
 
+var validFaviconRegex = regexp.MustCompile(`([^/]+\.js|favicon.ico)`)
```

Comment:
inludes js files also, not just favicon so the name `validFaviconRegex` is not correct